Docker/github_ci_dockerfile

```diff
@@ -81,6 +81,9 @@ ENV CPATH=/triqs/include:${CPATH} \
     LD_LIBRARY_PATH=/triqs/lib:${LD_LIBRARY_PATH} \
     PYTHONPATH=/triqs/lib/python3.10/site-packages:${PYTHONPATH} \
     CMAKE_PREFIX_PATH=/triqs/share/cmake:${CMAKE_PREFIX_PATH} \
+    CMAKE_BUILD_PARALLEL_LEVEL=${NCORES} \
+    CTEST_OUTPUT_ON_FAILURE=1 \
+    CTEST_PARALLEL_LEVEL=${NCORES} \
```

Comment:
I think github action runners have only two cores right? Maybe we should reduce `ARG NCORES=2` then. Right now it is 8 in all dockerfiles.

Reply:
is this overwriting the `NCORES` ARG in the Dockerfile, otherwise it will still use 8 cores for the build no?

Reply:
Yes, this is how `ARG` works. https://docs.docker.com/engine/reference/builder/#arg

Reply:
okay got it. Did not now the workflow passes this into the docker build. makes sense.